PRESUBMIT.py

```diff
@@ -499,4 +499,37 @@ def _IsJavaFile(input_api, file_path):
         ret.append(output_api.PresubmitError(msg))
     return ret
 
+
+@chromium_presubmit_overrides.override_check(globals())
+def CheckTodoBugReferences(_original_check, input_api, output_api):
+    """Checks that bugs in TODOs use updated issue tracker IDs."""
+
+    files_to_skip = [
+        'PRESUBMIT_test.py', r"^third_party/rust/chromium_crates_io/vendor/.*"
+    ]
+
+    def _FilterFile(affected_file):
+        return input_api.FilterSourceFile(affected_file,
+                                          files_to_skip=files_to_skip)
+
+    # Check for bug link in TODO comments.
+    pattern = input_api.re.compile(r'.*\bTODO\([^\)0-9]*([0-9]+)\).*')
```

Comment:
This pattern doesn't seem quite right - what about:


```suggestion
    pattern = input_api.re.compile(r'.*\bTODO\((\w+)\).*')
```

Reply:
^ this would match
```
// TODO(fallaciousreasoning): foo
but not
// TODO(https://github.com/brave/brave-browser/issues/1234): foo
```

Reply:
Thanks Jay for this.